src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Bulldozer bulldozer = new Bulldozer();
+        Excavator excavator = new Excavator();
+        Truck truck = new Truck();
```

Comment:
let's look through the checklist (common mistakes doc) - are all points from it implemented in the solution?

Reply:
done